libqtile/ffi_build.py

```diff
@@ -139,6 +140,17 @@
     gchar *
     g_markup_escape_text(const gchar *text,
                          gssize length);
+
+    // https://developer.gnome.org/pango/stable/pango-Cairo-Rendering.html
+    PangoCairoFontMap *
+    pango_cairo_font_map_get_default (void);
+    void
+    pango_cairo_font_map_set_default (PangoCairoFontMap *fontmap);
+    void
+    pango_cairo_font_map_set_resolution (PangoCairoFontMap *fontmap,
+                                         double dpi);
+
+    void free(void *);
```

Comment:
Why is the `free()` needed?

Reply:
Oh, this is me testing and being dumb, let me drop that, thanks.